Add GTN Event pages (#4890)
galaxyproject/training-material: events/2024-04-01-example-event.md

---
layout: event
title: My Training Event Title
draft: true  # will hide your event from the GTN events list, remove once you are ready to announce your event

description: |
    Short one or two line description of the event

cover-image: assets/images/gat.png
cover-image-alt: GTN Logo on a spiral galaxy background with text galaxy admin training

organizers: # GTN contributors or funders, must be defined in CONTRIBUTORS.yaml
- shiltemann
- hexylena

tags: [Topic 1, Topic 2, 5-day course]

date_start: 2024-04-01
date_end: 2024-04-02 # optional, if event is more than one day

> location: Example street 42, City, Country

Reviewer: ```

    "location": {
        "@type": "Place",
        "name": "University of Manchester",
        "address": {
            "@type": "PostalAddress",
            "streetAddress": "581 Oxford Road",
            "addressCountry": "United Kingdom",
            "addressRegion": "Manchester",
            "postalCode": "M13 9PL"
        },
        "geo": {
            "@type": "GeoCoordinates",
            "latitude": "-12.203029",
            "longitude": "42.203021"
        }
    },
```

I don't think we should include `geo`, if we really wanted that we could go back and geocode them with [nominatim](https://nominatim.openstreetmap.org/search.php?street=581+oxford+road&city=manchester&postalcode=M13+9PL&format=jsonv2) or similar later.

cost: free
audience: This event is intended for PhD students interested in Genomics. A basic knowledge in R is useful but not required.
contact_email: organizers@example.com

> async: false # if asynchronous, we will not add time columns to program

Reviewer: ```suggestion
async: false # if asynchronous, we will not display the time columns on the program
```

registraton:
- link:
  deadline:

feedback:
- link:
  deadline:

program:
  - section: "Monday: Introduction"
    description: Short description of the program.
    tutorials:
      - name: galaxy-intro-short
        topic: introduction
        time_start: "09:00"
        time_end: "10:00"
      - name: data-manipulation-olympics
        topic: introduction
        time_start: "10:00"
      - type: custom
        name: Coffee Break
        time_start: "10:30"
      - name: data-manipulation-olympics
        topic: introduction
        time_start: "10:45"
        time_end: "12:00"
      - name: ansible-galaxy
        topic: admin
        time_start: "13:00"
      - type: custom
        name: Custom Session
        time_start: "16:00"
        time_end: "17:00"
        description: |
          Description of the custom session here, in markdown, can add
          [links](https://example.com) if needed
      - type: custom
        name: Wrap-up & Drinks

>         time_start: "17:00"

Reviewer: I suppose some folks will want am/pm. Can we think of anything functional we want to do with the time? (calculating anything like course length or something?) Otherwise maybe we just have `time: "10am - 12pm", just a single plain text field for them to enter whatever they want.

cost: free # Or, e.g. 150 EUR, must be space separated, must include a currency in ISO 4217 format
audience: This event is intended for PhD students interested in Genomics. A basic knowledge in R is useful but not required.
contact_email: organisers@example.com
async: false # if asynchronous, we will not display the time columns on the program
mode: online # In-person
registration:
  link: https://example.org
  deadline: 2024-01-01

# Location of the event
# For online events, only the 'name' fields needs to be filled in
location:
  name: Erasmus Medical Center    # can be e.g. "Online" for online
  address: Dr. Molewaterplein 40
  city: Rotterdam
  country: The Netherlands
  #region: # optional
  postcode: 3015 GD
  geo:
    lat: 51.9109324
    lon: 4.4680514

# People involved, organisers, speakers, funders, etc
# Must be defined in CONTRIBUTORS.yaml file
contributions:
  organisers:
  - shiltemann
  - hexylena
  instructors:
  - bebatut
  - fpsom
  funding:
  - gallantries

# Galaxy and other infrastructure that will be used for your event.
# This will be used to create the setup instructions for participants
infrastructure:

>   tiaas: true    # tiaas = Training Infrastructure as a Service, and can be requested (for free) from all major Galaxies

Reviewer: should mention that there are convenient links in the 'instructor zone'
